spec/chewy/fields/base_spec.rb

```diff
@@ -413,16 +413,6 @@
           else
             Country.has_many :cities, order: :id
```

Comment:
Let's rewrite this to better correspond to reality. We have only one adapter now and it's always mandatory: we don't have any spec, that we don't want to run because of another adapter. Also we don't support `ActiveSupport` prior to 5.2.